Tools/autotest/sim_vehicle.py

```diff
@@ -911,7 +939,7 @@ def generate_frame_help():
                      help="Override simulation model to use")
 group_sim.add_option("", "--use-dir",
                      type='string',
-                     default=None,
+                     default='.',
```

Comment:
I am on phone and that change looks wrong. I will check later but it seems to launch sitl instance with root directory the one's where sim_vehicle.py is launched.

Reply:
. refers to the current working directory. I think this is consistent with the prior behavior. I made this change so that base_dir would always be set, even if the user did not specify --use-dir. This simplifies some of the code in start_vehicle and start_mavproxy. See: https://github.com/ArduPilot/ardupilot/pull/13405/files/b2ae81f729e513541d9f1d6bf84e1e0f324a9851#diff-460241a402d32677bcde6618e6e880deL1100-R1174